### DISCUSSION TITLE
Compatibilità con WordPress 6.2

### DISCUSSION THREAD

**Discussion by Dmonkeydemon** [fattureincloud/api#70] [Q&A] [answered]
Buongiorno, 
ho visto che c'è un aggiornamento per il plugin di WordPress, ma la versione 6.2 di WordPress sembra non compatibile con la versione 2.1.2 del plugin. Si può aggiornare lo stesso il plugin oppure a breve uscirà una versione testata con WordPress 6.2?

**Comment by valmoz** [comment, marked as answer]
Buongiorno @Dmonkeydemon 
posso chiederle a quale plugin si riferisce?

Grazie
Mauro

**Reply by valmoz** [reply]
Buongiorno @Dmonkeydemon restiamo in attesa di una sua risposta.
Senza sapere a quale plugin sta facendo riferimento non ci è possibile risponderle in maniera adeguata.

Grazie
Mauro

**Reply by valmoz** [reply]
Buongiorno @Dmonkeydemon 
Chiudo questa discussion per inattività.
Nel caso in cui avesse la necessità di riprendere la discussione potrà rispondere qui o eventualmente aprire una nuova discussion.

Grazie
Mauro
